src/errorHandling/errorObjectInterface.ts

```diff
@@ -0,0 +1,16 @@
+interface ErrorObjectInterface {
+    status:         number;
```

Comment:
The amount spaces after the `:` are unnecessary and adds extra work to the developer to ensure that every property has the necessary spaces after so types are on the same column. That is unnecessary, please just use 0 or 1 space after the `:`.
```suggestion
    status: number;
```